### PR TITLE
logs

### PR DESCRIPTION
**Pull request by StakanSoka**
do i need to use tests for such methods or maybe i wrote a bad logical programm?

### CODE REVIEW

File: src/main/java/lesson23/Shop.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    }
+
+    public Set<Order> startShopping() {
+        Set<Order> orders = new HashSet<>();
+        Order order;
+        boolean continues = true;
+
+        while (continues) {
+            order = takeNewOrder();
+            if (order != null) orders.add(order);
```
Comment (DHrytsuk) [inline src/main/java/lesson23/Shop.java:59]: Let's follow more common format
```
if (order != null)
{
orders.add(order);
}
```

File: src/main/java/lesson23/Shop.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+            return null;
+        }
+
+        return order;
+    }
+
+    public boolean askContinueQuestion() throws WrongAnswerException, NumberFormatException {
+        int answer;
+        int YES = 1;
```
Comment (DHrytsuk) [inline src/main/java/lesson23/Shop.java:39]: If you define it like a constant, then put on class level as constants

File: src/main/java/lesson23/Shop.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import java.util.HashSet;
+import java.util.Scanner;
+import java.util.Set;
+
+public class Shop {
+
+    private final Scanner scanner = new Scanner(System.in);
```
Comment (DHrytsuk) [inline src/main/java/lesson23/Shop.java:11]: It shouldn't be defined as a class level variable

Comment (StakanSoka) [reply src/main/java/lesson23/Shop.java:11]: But i use this in all my methods

Comment (DHrytsuk) [reply src/main/java/lesson23/Shop.java:11]: In this case you should separately define this in each method

File: src/main/java/lesson23/Shop.java (hunk 2 above)
Comment (StakanSoka) [reply src/main/java/lesson23/Shop.java:39]: I use such variables only in that method. Do i really need to put on them there?

Comment (DHrytsuk) [reply src/main/java/lesson23/Shop.java:39]: It seems, I've answered you on the previous lesson 😄

**Pull request merged**